.vale/styles/Vocab/word_list/accept.txt

```diff
@@ -78,3 +78,27 @@ AMIs
 [Cc]ancelation
 Brotli
 zlib
+datetime
+GZip
+videogame
+toc
+gcp
+faq
+synon
+npm
+html
+Maneu
+Hoxie
+Bleve
+Yonik
+Seeley
+MBs
+url
+masterKey
+SDK's
+Bardem
+LMDB's
+vogli
+belgium
```

Comment:
Should probably be capitalized wherever it is used, in which case it would probably not need to be added to this list?

Reply:
Can we change it to be capitalized in the search query and remove this from the list?

Reply:
We'll have to update the image: 
![Screen Shot 2023-02-13 at 14 33 01](https://user-images.githubusercontent.com/90181761/218434823-648884b5-fa7c-4d43-b965-d57032614b2b.png)

If we update the "Attribute" tab, we'll have to update the rest as well. This can be done in a separate PR as we have an [open issue for updating this example ](https://github.com/meilisearch/documentation/issues/851)

Reply:
Ok, not worth doing it just for this. I'll resolve this conversation, since it's related to the above one: finding a way to ignore alt text for linting would solve both problems.